README.md

```diff
@@ -43,11 +57,13 @@ Currently supported relations are
 
 - [Grafana](https://github.com/canonical/grafana-operator)
 - [Alertmanager](https://github.com/canonical/alertmanager-operator)
+- In addition this Prometheus charm does allow relations with any charm
+that supports the `prometheus_scrape` relation.
 
 ## OCI Images
 
-This charm by default uses the latest version of the Prometheus
-[Docker image](https://registry.hub.docker.com/r/prom/prometheus).
+This charm by default uses the latest version of the
+[ubuntu/prometheus](https://hub.docker.com/r/ubuntu/prometheus) image.
 
 ## Contributing
```

Comment:
This goes out of here and into a separate file, see https://warthogs.atlassian.net/browse/OP-205?atlOrigin=eyJpIjoiYjNjY2Q1NWU3NjRmNGJiYWI1MTA4ZTU3NzdjMWRjZWQiLCJwIjoiaiJ9

Reply:
Ack. Will do.